### ISSUE TITLE
BUG: PAYMENTOK not correctly parsing to payments.csv

### ISSUE THREAD

**Issue by leviable** [leviable/noso-go#21]
The PAYMENTOK response from the pool is not being correctly parsed to the payments.csv file

Example logs from noso-go at time of payment request:

```
<- PONG PoolData 5833 AE318AC52B68A7FD687E1677C3EFA2A4 11 0 103 148167860 34 204274
<- POOLSTEPS PoolData 5834 313D4A08E5E2CD7028641291B8DE4E1F 11 0 103 148167860 35 204274
-> PAYMENT
<- PAYMENTOK 1618891646 POOLIP Nm6jiGfRg7DVHHMfbMJL9CT1DtkUCF 2 5833 1.48153045 OR60v3w4j25pkl7mp2aaxa6l7g7hxqsdlfu86fkueh11tfyqg03z
```

Note that the miner balance was `148167860` or 1.48167860 NOS

Payment CSV file:
```
2021-04-19T23:07:26-05:00,noso.dukedog.io,Nm6jiGfRg7DVHHMfbMJL9CT1DtkUCF,Payment Request,5834,,
2021-04-19 23:07:26.573115 -0500 CDT m=+34358.112667405,noso.dukedog.io,Nm6jiGfRg7DVHHMfbMJL9CT1DtkUCF,Payment Response,5834,16.18891646,POOLIP
```
Looks like the epoch time, `1618891646` is getting parsed as the balance.

**Comment by leviable** [comment]
Still Some parsing issues, one more PR to fix

**Comment by leviable** [comment]
Fixed in v1.0.4
